rest-api/src/main/java/gov/cms/qpp/conversion/api/model/UnprocessedCpcFileData.java

```diff
@@ -0,0 +1,73 @@
+package gov.cms.qpp.conversion.api.model;
+
+import java.util.Date;
+
+/**
+ * Model to hold converted metadata to Unprocessed Cpc file data.
+ */
+public class UnprocessedCpcFileData {
+	private String fileId;
+	private String filename;
+	private String apm;
+	private Date conversionDate;
+	private Boolean validationSuccess;
+
+	/**
+	 * Constructor to transform metadata into unprocessed cpc file data
+	 *
+	 * @param metadata
+	 */
+	public UnprocessedCpcFileData(Metadata metadata) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document the parameter(s): metadata [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)